core/proto/core.proto

```diff
@@ -42,6 +44,8 @@ message Certificate {
   bytes der = 4;
   int64 issuedNS = 5;  // Unix timestamp (nanoseconds)
   int64 expiresNS = 6; // Unix timestamp (nanoseconds)
+  google.protobuf.Timestamp issued = 7;
```

Comment:
Field 8 technically isn't used in this message, should I use that number? From the proto3 docs, fields >= 16 take 2 bytes to encode, rather than 1 byte.

Reply:
Aha, field 8 was previously used and removed [here](https://github.com/letsencrypt/boulder/commit/7e994a12168aeaa4dda7c2ecc7ba92cc6c07e863#diff-80ea9561f1d738ba33d41a0776126deb77a355185a756cff4b77015bbd2a0e0f).